tsc/process/release.md

```diff
@@ -0,0 +1,20 @@
+**Release Process for OpenVDB**
+
+The following assumes that the current OpenVDB library version number is 6.0.0 and the new version number is 6.1.0. Adjust for the actual version numbers as appropriate.
+
+- [ ] Open a Jira "Release OpenVDB 6.1.0" ticket with "OpenVDB_6.1.0" as the Fix Version.
+- [ ] Update `openvdb/CHANGES` and `openvdb/doc/changes.txt` with release notes.
+- [ ] Update `openvdb/INSTALL` with the library version number and any directory hierarchy changes.
+- [ ] Push the above changes to `openvdb/master`, associating them with the Jira ticket created earlier, and verify that the Travis CI build runs successfully.
+- [ ] Draft a new [GitHub release](https://github.com/AcademySoftwareFoundation/openvdb/releases). Title it "OpenVDB 6.1.0" and tag it as `v6.1.0`.
+
+- [ ] Update `openvdb-website/contents/download/index.html` with the new version number (6.1.0) and release date, and `openvdb-website/contents/index.html` with a news item. Open the latter in a browser and check that the website renders correctly and that there are no broken links.
+- [ ] Build the documentation (both the `doc` and `pydoc` targets) and replace the contents of `openvdb-website/contents/documentation/doxygen/` with the output.  [_This step should be automated, and the thousands of files it generates should preferably not be committed to the repository._]
```

Comment:
I note that this doxygen documentation is for the last release. As part of the move to automate this, would it be desirable if we were to instead make this the doxygen for the current master branch? 

We're building all doc targets as part of our CI. If we implement the policy where we ensure that these are successful before a PR is merged, that means the doxygen would be always correct and in-sync with a fresh clone of the repo. I still think it could be useful to maintain doxygen for previous releases too though.

Reply:
I think we should make docs for all official releases available on the website (I do have archives of all of them), but certainly it wouldn't hurt to include an "in development" version as well.